docs/wiki/configurator/firmware-flasher-tab.md

```diff
@@ -4,11 +4,13 @@ sidebar_position: 21
 
 # Firmware Flasher Tab
 
-Here you update your firmware and select the features you require for you build.
+This tab is used to update the flight controller's Firmware.
+
+Since Betaflight 4.4, firmware updating is by an online build process. The user selects their flight controller, the code version to flash, and enables the features they want, then a custom firmware will be built online, downloaded, and installed onto the flight controller.
```

Comment:
```suggestion
Since Betaflight 4.4, updating the firmware is done via an online building process. The user selects their flight controller, the code version to flash, and enables the features they want, then a custom firmware will be built online, downloaded, and installed onto the flight controller.
```

Reply:
Done, thanks!

Reply:
Either my browser is not updating or it has not been committed :)